lib/collection/src/operations/conversions.rs

```diff
@@ -469,9 +469,8 @@ impl From<api::grpc::qdrant::OptimizersConfigDiff> for OptimizersConfig {
             memmap_threshold: optimizer_config.memmap_threshold.map(|x| x as usize),
             indexing_threshold: optimizer_config.indexing_threshold.map(|x| x as usize),
             flush_interval_sec: optimizer_config.flush_interval_sec.unwrap_or_default(),
-            max_optimization_threads: optimizer_config
-                .max_optimization_threads
-                .unwrap_or_default() as usize,
+            max_optimization_threads: optimizer_config.max_optimization_threads.unwrap_or(1)
```

Comment:
It feels like this new default to `1` is a new logic, I am unsure if the gRPC conversion should be responsible for it.
Why do we need this now?

Reply:
those changes are only needed for the possibility of doing patch release. It will be reverted in the next PR

Reply:
That's correct indeed.